resources/tiers.md

```diff
@@ -71,11 +71,30 @@ Whenever a anomaly is detected, a change can be submitted to the `integration-gu
 
 The other way around is also valid. A SDK can move to less important tiers. :shipit:
 
-#### Why SDKs are present in a particular tier?
+### Why SDKs are present in a particular tier?
 
 In general, because those SDKs have more and the most active users. By "active users," it means more activity in the repository, like contributions, watching (GitHub watchers), stargazers (GitHub stars), and forks (GitHub clones).
 
 Besides GitHub data, we track the importance of integration by constantly watching the data sent through the [telemetry](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html). Note that data is ANONYMOUS, but we know the used client by looking at the `User-Agent`/`X-Meilisearch-Client` headers.
 We also give more weight when we know a particular SDK has more Meilisearch Cloud customers.
 
 PS: Other principles that are not written here may be used to move SDKs around if the team thinks it is relevant.
+
+### When will the new features from a new Meilisearch release be released in the SDKs?
+
+There is a [guideline](https://github.com/meilisearch/integration-guides/blob/main/resources/pre-release-week.md) the Integrations Team follows for every pre-release. Take a look at it to have more info.
+
+Since every pre-release has its central issue ([like this one](https://github.com/meilisearch/integration-guides/issues/251)), and this issue describes every planned implementation the team will do in the weeks before the release, the community can expect that this plan will be achieved.
+
+But if, for some reason, the team does not finish it on time, it will still be the top priority to finish it before anything else. In an event of having only part of the features available to be released in the D-Day, the team will evaluate how long it will take to finish the remaining, so it means:
```

Comment:
```suggestion
But if, for some reason, the team does not finish implementing on time, it still is the top priority to finish it before anything else. If only parts of the features are available on the release D-Day, the team will evaluate how long it will take to finish them. Depending on the integration tier, and the time it would take the team to finish, we will release the integration with the partial implementations. These are the delays we accept per tier before releasing:
```

Just changed from "us" to "the team".